<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="DESKTOP-1HG8JJK" failures="1" tests="1" name="Default test(failed)" time="0.098" errors="0" timestamp="2021-07-15T17:45:59 IST">
  <testcase classname="listeners.RetryFailedTests" name="test2" time="0.017">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at listeners.RetryFailedTests.test2(RetryFailedTests.java:14)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- test2 -->
  <testcase classname="listeners.RetryFailedTests" name="test1" time="0.0">
    <ignored/>
  </testcase> <!-- test1 -->
</testsuite> <!-- Default test(failed) -->
